### PR TITLE
fix_hw_wallets_build

### PR DESCRIPTION
**Pull request by zebra-lucky**
- Fix win/osx builds to properly import all dynamic imports
- Small fixes to trezor plugin, install python-trezor from github repo
- Fix some packages version (need by python-trezror)
- Add console-electrum-dash.exe variant to win build to watch posssible runtime traces.

### CODE REVIEW

File: plugins/trezor/plugin.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def _try_hid(self, device):
             return self.hid_transport(pair)
         except BaseException as e:
             # see fdb810ba622dc7dbe1259cbafb5b28e19d2ab114
```
Comment (akhavr) [inline plugins/trezor/plugin.py]: Should we remove this completely?

File: plugins/trezor/plugin.py [hunk 2]
```diff
@@ -106,7 +106,9 @@ def _try_bridge(self, device):
             return None
 
     def create_client(self, device, handler):
-        transport = self._try_bridge(device) or self._try_hid(device)
+        # disable bridge because it seems to never returns if keepkey is plugged
+        #transport = self._try_bridge(device) or self._try_hid(device)
```
Comment (akhavr) [inline plugins/trezor/plugin.py]: Should we remove the commented out line and stick just to `hid`?

File: plugins/trezor/plugin.py (hunk 1 above)
Comment (zebra-lucky) [reply plugins/trezor/plugin.py]: It's still in electrum master branch

File: plugins/trezor/plugin.py (hunk 2 above)
Comment (zebra-lucky) [reply plugins/trezor/plugin.py]: It's still in electrum master branch (only _try_hid)

File: plugins/trezor/plugin.py (hunk 1 above)
Comment (akhavr) [reply plugins/trezor/plugin.py]: I suggest to remove both `raise` and the comment and submit it to the upstream.  If it's necessary for some reason, they'll let us know.

File: plugins/trezor/plugin.py (hunk 2 above)
Comment (akhavr) [reply plugins/trezor/plugin.py]: Same as above: remove the comment by leaving only `_try_hid` here and submit this patch upstream.

**Pull request merged**